pkgs/tools/system/gptman/default.nix

```
@@ -1,23 +1,31 @@
-{ lib, stdenv, fetchFromGitHub, rustPlatform, libiconv }:
+{ lib, stdenv, fetchFromGitHub, rustPlatform, libiconv, testers, gptman }:
 
 rustPlatform.buildRustPackage rec {
   pname = "gptman";
-  version = "0.8.3";
+  version = "1.0.1";
 
   src = fetchFromGitHub {
-    owner = "cecton";
+    owner = "rust-disk-partition-management";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-hI3F1E1vdbNDEeJ4FrU0EvR0t64svzUIpI6zaf0CquM=";
+    hash = "sha256-sDRnvF/XPXgWIIIrOmnEuktP8XvZxPahF2n4h8RCX+o=";
   };
 
-  cargoSha256 = "sha256-3PRGPZGymccRo9dtQZgMMEL29x+GiUkTzgc8uAB/ocQ=";
+  cargoHash = "sha256-voslPSh7n31cGTKaayKXomgiXWVTutuc4FxfnZUDejc=";
 
   buildInputs = lib.optional stdenv.isDarwin libiconv;
 
+  cargoBuildFlags = [ "--all-features" ];
+
+  cargoTestFlags = [ "--workspace" "--all-features" ];
```

This should be enough
```suggestion
  buildFeatures = [ "cli" ];
```

---

I'm OK with enabling `cli` feature only. But I think it's better to stay in line with upstream. See https://github.com/rust-disk-partition-management/gptman/blob/b0c3567c19c1b23f95b5de665a58d611301e3706/.github/workflows/release.yml#L25.

---

that's the only feature